backend-rust/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Fixed
+
+- Validation on rows affected on bakers only occurs after protocol version 7 is in effect
```

Comment:
Protocol 6 or 7, can we have a very short explanation why we fix that.

Reply:
Arent this a changelog? This should describe the changes made. It really sounds like a lot of noice if we in a changelog are to justify all changes made.